integration/js/SendingAudioFailureAndRecoveryTest.js

```diff
@@ -0,0 +1,38 @@
+
+const {TestUtils} = require('./node_modules/kite-common');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const {OpenAppStep, AuthenticateUserStep, JoinMeetingStep, SelectNoAudioInputStep, PlayPrerecordedSpeechStep} = require("./steps");
+const {UserAuthenticationCheck, SendingAudioFailureAndRecoveryCheck, UserJoinedMeetingCheck} = require("./checks");
+const {v4: uuidv4} = require("uuid");
+
+class SendingAudioFailureAndRecoveryTest extends SdkBaseTest {
+    constructor(name, kiteConfig) {
+        super(name, kiteConfig, "Sending Audio Failure and Recovery Test");
+    }
+
+    async runIntegrationTest() {
+        this.numberOfParticipant = 1;
+        const session = this.seleniumSessions[0];
+        const attendeeID = uuidv4();
+
+        await OpenAppStep.executeStep(this, session);
```

Comment:
Hmmm this uses browser demo instead of custom integration test app right?...this is fine but ideally we wouldn't be using the demo app for this.

Reply:
I was unaware that we have a custom integration test app, let me look into that

Reply:
Yes we do have to specifically test the video APIs here:
https://github.com/aws/amazon-chime-sdk-js/tree/main/integration/js/app

For this test, may it is good to add an audio demo and test them under it? @ltrung any suggestions? Then may be we also do not have to block on the firefox issue with no audio at least for the main PR merge.

Reply:
Ideally, we want separate demo for video and audio testing and not keep adding test stuffs in the kitchen sink demo. In this case, however, I don't think it matters that much as it is just an event check and there is not much changes we make to the demo.